<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kalkulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>50</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="tombol1">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol2">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol3">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol4">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombolce">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="tombol5">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol6">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol7">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol8">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombolplus">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="tombol9">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol0">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombolbagi">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombolkali">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombolminus">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="tombolequ">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tombol9_3">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
